docs/development/docs.rst

```diff
@@ -10,27 +10,48 @@ The docs are kept in the ``docs/`` directory at the top of the source tree.
    However, if there is going to be periodic/bigger contributions, it may be a
    good idea to install the Sphinx requirements to build our docs.
 
+Contributing through the Github UI
+----------------------------------
+
+If you're making small changes to the documentation, you can verify those changes through the documentation generated when you open a PR and can be accessed using the Github UI.
+
+1. click the checkmark next to your commit and it will expand to have multiple options
+2. right-click the "details" link next to the "docs/readthedocs.org:docs" item
+3. navigate to the section of the documentation you worked on to verify your changes
+
+Contributing from your local machine
+------------------------------------
+
+If you're making large changes to the documentation, you'll want to verify those changes locally before pushing upstream.
+
+1. follow the instructions to `Set up your environment <https://docs.readthedocs.io/en/stable/development/install.html#set-up-your-environment>`__
+2. navigate to ??? to view your development version of the Read The Docs documentation
+3. as long as all your containers are up as shown in step 6 of `Set up your environment <https://docs.readthedocs.io/en/stable/development/install.html#set-up-your-environment>`__, your changes should be reflected automatically on each save
```

Comment:
I think the "as long as..." clarification is perhaps a bit redundant.

Also, "changes should be reflected automatically on each save" might be misleading: builds don't automatically trigger on disk changes, nor on new local commits (just tested it). Therefore, we should expand these instructions a bit and explain that we would need to rebuild the docs for every change.